src/wallet/wallet.h

```diff
@@ -348,6 +349,8 @@ class CWallet final : public WalletStorage, public interfaces::Chain::Notificati
      */
     static bool AttachChain(const std::shared_ptr<CWallet>& wallet, interfaces::Chain& chain, const bool rescan_required, bilingual_str& error, std::vector<bilingual_str>& warnings);
 
+    static int GetDefaultNextResend() { return GetTime() + (12 * 60 * 60) + GetRand(24 * 60 * 60); }
```

Comment:
Any reason to truncate the result?

Reply:
What do you mean with truncating here? I just moved the existing logic into a new function, that on its own shouldn't be any behavior change.